sonarqube-ide-visualstudio-roslyn-plugin/src/main/java/org/sonarsource/sonarlint/visualstudio/roslyn/SqvsRoslynSensor.java

```diff
@@ -66,13 +67,34 @@ private static void handle(SensorContext context, RoslynIssue roslynIssue) {
           .forRule(ruleKey)
           .at(createLocation(newIssue, roslynIssue.getPrimaryLocation(), diagInputFile));
         handleSecondaryLocations(context, roslynIssue, newIssue);
-        // TODO by https://sonarsource.atlassian.net/browse/SLVS-2492 handle quickfixes once they are returned from the server
-        /* handleQuickFixes(context, roslynIssue, newIssue); */
+        handleQuickFixes(context, roslynIssue, newIssue);
         newIssue.save();
       }
     }
   }
 
+  private static void handleQuickFixes(SensorContext context, RoslynIssue roslynIssue, NewIssue newIssue) {
+    if (roslynIssue.getQuickFixes().isEmpty()){
+      return;
+    }
+    var filePath = Paths.get(roslynIssue.getPrimaryLocation().getFilePath());
+    var file = findInputFile(context, filePath);
+    if (file == null){
+      return;
+    }
+
+    newIssue.setQuickFixAvailable(true);
```

Comment:
This has no effect in the SLCORE context, you could remove it

Reply:
removing it